corehq/apps/data_interfaces/models.py

```diff
@@ -587,7 +587,7 @@ class MatchPropertyDefinition(CaseRuleCriteriaDefinition):
 
     property_name = models.CharField(max_length=126)
     property_value = models.CharField(max_length=126, null=True)
-    match_type = models.CharField(max_length=15)
+    match_type = models.CharField(max_length=25)
```

Comment:
Can the values of `MATCH_DAYS_` constants be changed to avoid the migration?

Since none of the too long values have ever been saved in the database, it seems like there is no backward compatibility concern. One possible example:

```py
    MATCH_DAYS_LESS_THAN = 'DAYS_BEFORE'
    MATCH_DAYS_LESS_OR_EQUAL = '<='

    # True when today >= (the date in property_name + property_value days)
    MATCH_DAYS_GREATER_THAN = '>'
    MATCH_DAYS_GREATER_OR_EQUAL = 'DAYS'
```

Reply:
Rebased the PR to remove the migration. Adopted a version of this that just uses 'LTE' and 'GT' rather than the symbols, as it felt more consistent with the other constants